Rubberduck.Core/Common/WinAPI/User32.cs

```diff
@@ -45,8 +45,9 @@ public static class User32
         [return: MarshalAs(UnmanagedType.Bool)]
         public static extern bool UnregisterHotKey(IntPtr hWnd, IntPtr id);
         
+        // TODO used to be internal. InternalsVisibleTo should expose this, right?
```

Comment:
Comment still needed? I believe `InternalsVisibleTo` would expose this, but the `extern` just *might* make it behave differently. Do the tests still run?